clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/containers/KubernetesCluster.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.it.containers;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.springframework.util.FileCopyUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.yaml.snakeyaml.Yaml;
+
+public class KubernetesCluster extends GenericContainer {
+
+  private static final String DOCKER_IMAGE = "rancher/k3s:v1.17.11-k3s1";
+  private static final String KUBECFG_IN_CONTAINER = "/etc/rancher/k3s/k3s.yaml";
+
+  private static final Map<String, KubernetesCluster> instances = new HashMap<>();
+
+  private Path kubecfgPath;
+  private final String accountName;
+
+  public static KubernetesCluster getInstance(String accountName) {
+    if (instances.get(accountName) == null) {
```

Comment:
Really minor, feel free to ignore, but could be done as:
```
return instances.computeIfAbsent(accountName, KubernetesCluster::new);
```
which might be a tiny bit clearer.